src/npm/root-utils.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+// this file will be added to the root after build
+// he allows users to require utils package as:
```

Comment:
```suggestion
// This allows users to require utils package as:
```

Reply:
Why are we using “he”?

Reply:
Im just waiting other review to change my code. Sorry for that by the way. I use French for construct my sentences before write it in English ahhaah.

Reply:
No problem, @tony-go. My native language is Portuguese - which is also a strongly gender-oriented language - and this is something that can happen 😄